### ISSUE TITLE
feature: local-ai subcommands/separate CLI to operate the API

### ISSUE THREAD

**Issue by mudler** [go-skynet/LocalAI#816]
I think would be quite useful to have subcommands in the CLI that acts as a client to the API allowing to e.g:

- Install/setup models with the gallery
- Run inference quickly
- TTS
- Text to audio

Another great good first issue for those that want to start contributing to the project

**Comment by jespino** [comment]
I want to take a look into this :)

**Comment by mudler** [comment]
github automation.. maybe better to keep this open to track and make sub-items

**Comment by jespino** [comment]
By the way, I'm not doing "API calls to a running localAI", I'm adding a command line interface that skips the server process entirely.

**Comment by mudler** [comment]
gotcha, maybe makes sense to have a local-ai client <> subcommand as well in order to run against apis. I see that helpful especially to load models from galleries

**Comment by jespino** [comment]
Actually, I see a lot of value on having the API based client separated from the main binary, because compiling the main binary is hard, but compiling a tiny client library + command line that at the end of the day is just a bunch of HTTP requests should be fairly easy.

I totally see people running the LocalAI using docker compose and accessing it from the command line using something like `localai-cli`.

**Comment by mudler** [comment]
right! that could also sit nearby here and have a separate make target or a separate repository.. I have no strong opinion here, I'm fine with both
